mhctools/cli/args.py

```diff
@@ -71,6 +79,15 @@
     "mhcflurry": MHCflurry,
 }
 
+# The versioned program names will be tried first, and if they don't exist in the path, we'll
+# fall back to the standard install program names.
+versioned_mhc_predictor_program_names = {
+    "netmhcpan28": ["netMHCpan-2.8", "netMHCpan"],
```

Comment:
I think this will create pretty subtle bugs around creating a e.g. NetMHCpan28 predictor whose commandline predictor is actually 4.0